src/test/resources/goal-output/commons-math3-3.6.1/org.apache.commons.math3.fraction.BigFraction_goal.json

```diff
@@ -814,7 +814,7 @@
           "isArray": false
         },
         "comment": "if the fraction to divide by is zero",
-        "condition": "args[0].numerator.signum()==0"
+        "condition": "args[0].abs()==0"
```

Comment:
I'll change it to the second one, sounds better that a function, but I thought that the use of abs() to get the value of the object was valid. 

Should I do the same from now on and use that kind of comparisons instead of calling functions?